docs/api/generator/password.md

```diff
@@ -49,3 +50,27 @@ With default values you would get something like:
 ZRv-k!y6x/V"29:43aErSf$1
 Vk9*mwXE30Q+>H?lY$5I64_q
 ```
+
+## Encoding Examples
+
+The password generator supports different encoding formats for the output:
+
+```yaml
+{% include 'generator-password-encoding-examples.yaml' %}
+```
+
+### Encoding Output Examples
+
+For the same password `Test>>Pass??word`, the different encodings would produce:
+
+- **raw** (default): `Test>>Pass??word` (original password string)
+- **base64**: `VGVzdD4+UGFzcz8/d29yZA==` (standard base64)
+- **base64url**: `VGVzdD4-UGFzcz8_d29yZA==` (URL-safe base64)
+- **base32**: `ORSXG5BRGIYTEMJQGQYQ====` (base32 encoding)
+- **hex**: `546573743e3e506173733f3f776f7264` (hexadecimal encoding)
```

Comment:
While I know this probably is enough for your use case, I just wanted to confirm - this approach means several hex strings are never going to be produced (like `0123456789abcdef`) because they're not part of the alphabet we use.